### PR TITLE
[NETBEANS-54] Module Review i18n

### LINKED ISSUE #74: Cleanup NetBeans license header for module i18n
Review and cleanup the license header in i18n

Oracle donated NetBeans to the Apache Software Foundation.

Your task now is to *replace* the old license header that has Oracle (and/or Sun Microsystems
and/or NetBeans) in it with the standard Apache Software Foundation license header.

For an empty file that supports headers you must *add* the Apache license header.

Note that you have to use the proper license header for Java, .properties or HTML files.
Look at the nearby files for templates and copy-paste the license header.

When done commit the changes with a commit message that starts with first line
"[NETBEANS-54] Module Review <module>" and a message body describing your review.

See a longer explanation about what we are doing here: https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review

Please announce in a comment when you have started work.

Files that need to be looked at in i18n:
```
i18n/src/org/netbeans/modules/i18n/jsp/Bundle.properties
i18n/test/qa-functional/data/projects/ProjectI18n/build.xml
i18n/test/unit/src/org/netbeans/modules/i18n/java/resources/JFrame.txt

```

### PR DESCRIPTION
**Pull request by oktak**
I also have a look on the 2 files below, no old license header. Fixes #74 

i18n/test/qa-functional/data/projects/ProjectI18n/build.xml
i18n/test/unit/src/org/netbeans/modules/i18n/java/resources/JFrame.txt

### CODE REVIEW

File: i18n/src/org/netbeans/modules/i18n/jsp/Bundle.properties [hunk 1]
```diff
@@ -1,38 +1,19 @@
...
-# Version 2 license, then the option applies only if the new code is
-# made subject to such option by the copyright holder.
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
 #
 # Contributor(s):
```
Comment (emilianbold) [inline i18n/src/org/netbeans/modules/i18n/jsp/Bundle.properties]: The `#Contributors` line is part of the previous license header. Please remove that line too.

**Review by emilianbold — CHANGES_REQUESTED** [review]

**Comment by emilianbold** [comment]
> i18n/test/unit/src/org/netbeans/modules/i18n/java/resources/JFrame.txt

You may also add a license header to this file. Seem to be a Java file with a .txt extension.

>i18n/test/qa-functional/data/projects/ProjectI18n/build.xml

You may leave this as it is. build.xml files will be handled all at once at a later time.

File: i18n/src/org/netbeans/modules/i18n/jsp/Bundle.properties (hunk 1 above)
Comment (oktak) [reply i18n/src/org/netbeans/modules/i18n/jsp/Bundle.properties]: I see, let me remove this and add the license header for the file below.

**Comment by oktak** [comment]
@emilianbold I've a new commit for the changes requested.

**Review by emilianbold — APPROVED** [review]

**Pull request merged**